### ISSUE TITLE
MSVC crashes in Debug mode when joining a map

### ISSUE THREAD

**Issue by sjrc6** [ddnet/ddnet#9277]
Unhandled exception at 0x00007FF7B500F1E7 in DDNet.exe: 0xC00000FD: Stack overflow (parameters: 0x0000000000000001, 0x000000782FE03000).
```
DDNet.exe!__chkstk() Line 109	Unknown
DDNet.exe!CClient::SendMsg(int Conn, CMsgPacker * pMsg, int Flags) Line 171	C++
DDNet.exe!CClient::SendReady(int Conn) Line 246	C++
DDNet.exe!CClient::ProcessServerPacket(CNetChunk * pPacket, int Conn, bool Dummy) Line 1542	C++
DDNet.exe!CClient::PumpNetwork() Line 2521	C++
DDNet.exe!CClient::Update() Line 2600	C++
DDNet.exe!CClient::Run() Line 3174	C++
DDNet.exe!main(int argc, const char * * argv) Line 4863	C++
DDNet.exe!invoke_main() Line 79	C++
DDNet.exe!__scrt_common_main_seh() Line 288	C++
DDNet.exe!__scrt_common_main() Line 331	C++
DDNet.exe!mainCRTStartup(void * __formal) Line 17	C++
kernel32.dll!00007ffe01e17374()	Unknown
ntdll.dll!00007ffe0267cc91()	Unknown
```

**Comment by sjrc6** [comment]
offending commit: https://github.com/ddnet/ddnet/commit/1b6e5a9295787f831339262a64bd7f805e966819

**Comment by Robyt3** [comment]
Should be fixed by #9272.
